sci-libs/buddy/buddy-2.4-r1.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit fortran-2
+
+DESCRIPTION="Binary Decision Diagram Package"
+HOMEPAGE="https://sourceforge.net/projects/buddy/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="buddy"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux ~ppc-macos"
+IUSE="doc examples static-libs"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-gcc43.patch
+	"${FILESDIR}"/${P}-gold.patch
+)
+
+DOCS=(
+	"${S}"/doc/tech.txt
+)
+
+src_configure() {
+	econf $(use_enable static-libs static)
+}
+
+src_install() {
+	default
+
+	einstalldocs
+
+	use doc && docinto /usr/share/doc/${PF}/ps && dodoc doc/*.ps
```

Comment:
Not sure if there is a solution for the DOCS approach which changes the destination of the doc files